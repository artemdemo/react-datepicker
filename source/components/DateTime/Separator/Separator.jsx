import React from 'react';

import './Separator.less';

const Separator = () => {
    return (
        <hr className='datetime-separator' />
    );
};

Separator.displayName = 'Separator';

export default Separator;
