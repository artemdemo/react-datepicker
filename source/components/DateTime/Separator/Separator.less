.datetime-separator {
    border: transparent;
    border-bottom: 1px solid #c3c3c3;
}
