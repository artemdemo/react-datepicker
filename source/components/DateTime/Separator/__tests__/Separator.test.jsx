import React from 'react';
import renderer from 'react-test-renderer';
import Separator from '../Separator';

describe('<Separator />', () => {
    it('simple render', () => {
        const tree = renderer.create(
            <Separator />,
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
