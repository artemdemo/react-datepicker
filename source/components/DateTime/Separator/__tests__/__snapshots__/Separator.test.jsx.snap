// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Separator /> simple render 1`] = `
<hr
  className="datetime-separator"
/>
`;
